src/lib.rs

```diff
@@ -1,46 +1,8 @@
-// Copyright 2023 the Velato Authors
```

Comment:
We probably should have some usage docs here or in the readme

Reply:
This can be a followup task addressed in #14